### PR TITLE
CAIP-26 | Blockchain reference for Tezos

### PR DESCRIPTION
**Pull request by stanly-johnson**
This PR adds chain-id for Tezos to the list.

Let me know if a seperate issue has to be created to track this PR

### CODE REVIEW

**Comment by stanly-johnson** [comment]
Related : https://gitlab.com/tezos/tezos/-/issues/1029

**Review by ligi — APPROVED** [review]
LGTM

File: CAIPs/caip-26.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Specification
+
+### Tezos Namespace
+
+The namespace "tezos" refers to the Tezos open-source blockchain platform.
+
+#### Reference Definition
+
+The definition for this namespace will use the `genesis-block-hash` as an indentifier for different Tezos chains. The method for calculating the chain Id is as follows
```
Comment (webmaster128) [inline CAIPs/caip-26.md:34]:
```suggestion
The definition for this namespace will use the `genesis-block-hash` as an indentifier for different Tezos chains. The method for calculating the chain ID is as follows
```

File: CAIPs/caip-26.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+```
+tezosB58CheckEncode('Net',
+  firstFourBytes(
+    blake2b(msg = tezosB58CheckDecode('B', genesisBlockHash),
+            size = 32)))
+```
+
+## Rationale
+
+Blockchains in the "tezos" namespace are identified by their Chain ID as mentioned in the Reference Definition Section.
```
Comment (webmaster128) [inline CAIPs/caip-26.md:44]:
```suggestion
Blockchains in the "tezos" namespace are identified by their chain ID as mentioned in the Reference Definition Section.
```

File: CAIPs/caip-26.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+#### Reference Definition
+
+The definition for this namespace will use the `genesis-block-hash` as an indentifier for different Tezos chains. The method for calculating the chain Id is as follows 
+```
+tezosB58CheckEncode('Net',
+  firstFourBytes(
+    blake2b(msg = tezosB58CheckDecode('B', genesisBlockHash),
+            size = 32)))
+```
```
Comment (webmaster128) [inline CAIPs/caip-26.md:40]: Hmm, four bytes is really small for avoiding collisions. Do you expect many tezos chains in the future?

I'm just curious. If the Tezos community is happy with this format, it's good for this CAIP as well.

File: CAIPs/caip-26.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+### Tezos Namespace
+
+The namespace "tezos" refers to the Tezos open-source blockchain platform.
+
+#### Reference Definition
+
+The definition for this namespace will use the `genesis-block-hash` as an indentifier for different Tezos chains. The method for calculating the chain Id is as follows 
+```
+tezosB58CheckEncode('Net',
```
Comment (webmaster128) [inline CAIPs/caip-26.md:36]: Is this "Net" from [Base58.Prefix.chain_id](https://gitlab.com/tezos/tezos/blob/e7612c5ffa46570cdcc612f7bcead771edc24283/src/lib_crypto/chain_id.ml#L108)?

File: CAIPs/caip-26.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+# Tezos DelphiNet (Current active testnet)
+tezos:NetXm8tYqnMWky1
+```
+
+## Links
+
+- [Implementation](https://gitlab.com/tezos/tezos/blob/e7612c5ffa46570cdcc612f7bcead771edc24283/src/lib_crypto/chain_id.ml)
+
+
```
Comment (webmaster128) [inline CAIPs/caip-26.md:66]:
```suggestion

```

**Comment by webmaster128** [comment]
> Related : https://gitlab.com/tezos/tezos/-/issues/1029

Could you add this link to the document, either in `discussions-to:` or to the links section in the bottom?

File: CAIPs/caip-26.md (hunk 4 above)
Comment (stanly-johnson) [reply CAIPs/caip-26.md:36]: Yes, [Reference](https://gitlab.com/tezos/tezos/-/blob/e7612c5ffa46570cdcc612f7bcead771edc24283/src/lib_crypto/base58.ml#L358)

File: CAIPs/caip-26.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+caip: 26
+title: Blockchain Reference for the Tezos Namespace
+author: Stanly Johnson (@stanly-johnson)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/36, https://gitlab.com/tezos/tezos/-/issues/1029
```
Comment (webmaster128) [inline CAIPs/caip-26.md:5]: This is YAML syntax, let's use this:

```suggestion
discussions-to: [https://github.com/ChainAgnostic/CAIPs/pull/36, https://gitlab.com/tezos/tezos/-/issues/1029]
```

Comment (webmaster128) [reply CAIPs/caip-26.md:5]: @ligi should we allow multiple links in this field?

**Review by webmaster128 — COMMENTED** [review]
LGTM

File: CAIPs/caip-26.md (hunk 3 above)
Comment (stanly-johnson) [reply CAIPs/caip-26.md:40]: Tezos is not meant for a multi chains (like cosmos or polkadot), if thats what you mean. Beyond that I agree with your point. Maybe someone from the core team can comment.

**Pull request merged**